functional/cypress-tests/cypress/integration/log_viewer.js

```diff
@@ -0,0 +1,57 @@
+import { logsViewer } from '../fixtures/Widgets';
+import { createWidget } from '../support/widget';
+
+const dashboardName = 'Welcome to Cogboard';
+
+describe('Logs Viewer', () => {
+  let widget;
+
+  before(() => {
+    cy.visit('/');
+    cy.login();
+    cy.openDrawer();
+    cy.chooseDashboard(dashboardName);
+    cy.clickAddWidgetButton();
+    widget = createWidget(logsViewer.name).configure(false, {
+      cols: 8,
+      rows: 2
+    });
+  });
+
+  beforeEach(() => {
+    cy.viewport(1920, 1080);
+  });
+
+  describe('Searchbar', () => {
+    it('Search icon is shown while there are less than 3 letters in input', () => {
```

Comment:
opis tych testów powinien mieć zawsze na uwadze ten przedrostek `it` na początku, na przykład:
`it('should show search icon...` albo `it('shows icon`